core/constants.py

```diff
@@ -48,3 +48,40 @@
 YEAR_CHOICES = []
 for year in range(2013, (datetime.now().year + 1)):
     YEAR_CHOICES.append((year, year))
+
+MAKE = 'Make'
+TYPE = 'Type'
+ASSET_CODE = 'Asset Code'
+CATEGORY = 'Category'
+SUB_CATEGORY = 'Sub-Category'
+MODEL_NUMBER = 'Model Number'
+SERIAL_NUMBER = 'Serial No.'
+ASSIGNED_TO = 'Assigned To'
+STATUS = 'Status'
+ASSET_MEMORY = 'Memory'
+VERIFIED = 'Verified'
+NOTES = 'Notes'
+STORAGE = 'Storage'
+ASSET_PROCESSOR_TYPE = 'Processor Type'
+YOM = 'YOM'
```

Comment:
Let's use these constants [here](https://github.com/AndelaOSP/art-backend/blob/20087a548d89ac318fd6dc8b60738531aaf6c5df/core/assets_import_helper.py#L39) as well, and wherever else these field names are used